src/cascadia/WindowsTerminal/WindowThread.h

```diff
@@ -17,7 +17,7 @@ class WindowThread : public std::enable_shared_from_this<WindowThread>
     int RunMessagePump();
     void RundownForExit();
 
-    winrt::Microsoft::Terminal::Remoting::Peasant Peasant();
+    uint64_t PeasantID();

```

Comment:
I made this change because it avoids exposing the innards of `WindowThread`. It's technically unrelated.

Reply:
LGTM love it